### ISSUE TITLE
add stats section

### ISSUE THREAD

**Issue by contra** [gulpjs/gulpjs.github.io#52]
Mockup:

<img width="1388" alt="screen shot 2017-01-17 at 4 17 41 pm" src="https://cloud.githubusercontent.com/assets/425716/22040342/83d09710-dcd0-11e6-807f-ac13545fcd96.png">

**Comment by SomethingNew71** [comment]
I can help with this if you are interested.  Is there an API endpoint that would provide this data?

**Comment by phated** [comment]
I don't think there's an API that gets both installs and plugins.  Installs would come from npm and plugins would come from npmsearch.  The companies and seconds saved would be static, I'm guessing.

**Comment by SomethingNew71** [comment]
Do you like the look of this? It's just mocked up with static data I did last night.

![screen shot 2017-10-26 at 3 14 12 pm](https://user-images.githubusercontent.com/4572799/32072369-58973fea-ba60-11e7-9d66-f97016447820.png)

**Comment by SomethingNew71** [comment]
And I have found the plugin count based on a search.

#### API for Related Plurgins - https://api-docs.npms.io/#api-Search-ExecuteSearchQuery

Probably could grab the total out of the response.

*Example:* `https://api.npms.io/v2/search?q=gulp`
*Results:*
```json
{
  "total": 11424,
  "results": [{
    "package": {
      "name": "gulp",
      "scope": "unscoped",
      "version": "3.9.1",
      "description": "The streaming build system",
      "date": "2016-02-08T18:50:16.472Z",
      "links": {
        "npm": "https://www.npmjs.com/package/gulp",
        "homepage": "http://gulpjs.com",
        "repository": "https://github.com/gulpjs/gulp",
        "bugs": "https://github.com/gulpjs/gulp/issues"
      },
      "author": {
        "name": "Fractal",
        "email": "contact@wearefractal.com",
        "url": "http://wearefractal.com/"
      },
      "publisher": {
        "username": "phated",
        "email": "blaine@iceddev.com"
      },
      "maintainers": [{
          "username": "contra",
          "email": "yo@contra.io"
        },
        {
          "username": "phated",
          "email": "blaine.bublitz@gmail.com"
        }
      ]
    },
    "flags": {
      "insecure": 2
    },
    "score": {
      "final": 0.7768170783638859,
      "detail": {
        "quality": 0.8562822998580506,
        "popularity": 0.8196480107850486,
        "maintenance": 0.6658730989477251
      }
    },
    "searchScore": 100000.78, 
    {
      ....more packages.....
    }
  }]
}
```

**Comment by phated** [comment]
Closed by #75 thanks to @SomethingNew71
